Alpha database migration guide inacuracies

### Existing documentation URL(s)

https://developers.cloudflare.com/d1/platform/alpha-migration/

### What changes are you suggesting?

In step 4, it references a table named `_cf_KV` in the database dump. It was by a different name in my dump: `d1_kv`. 
If there are multiple table names, maybe updating it to list all variants would be appropriate. The schema was the same as described in step 4.

In step 6, it references a flag `--remote`, for which I receive the following from wrangler.
```
✘ [ERROR] Unknown argument: remote
```

Removing the `--remote` flag allowed me to complete step 6. 
Wrangler has a `--local` flag and not a `--remote` flag in the `wrangler d1 execute <name>` command help text.

### Additional information

_No response_

--- a/content/d1/platform/alpha-migration.md
+++ b/content/d1/platform/alpha-migration.md
@@ -16,6 +16,12 @@ D1's open beta launched in October 2023, and newly created databases use a diffe
 
 This guide will instruct you to recreate alpha D1 databases on our production-ready system.
 
+## Prerequisites
+
+1. You have the [`wrangler` command-line tool](/workers/wrangler/install-and-update/) installed
+2. You are using `wrangler` version `3.33.0` or later (released March 2024) as earlier versions do not have the [`--remote` flag](`/d1/platform/changelog/#change-in-wrangler-d1-execute-default) required as part of this guide
+3. An 'alpha' D1 database. All databases created before July 27th, 2023 ([release notes](/d1/platform/changelog/#new-default-storage-subsystem)) use the alpha storage backend, which is no longer supported and was not recommended for production.
+
 ## 1. Verify that a database is alpha
 
 ```sh

--- a/content/d1/platform/alpha-migration.md
+++ b/content/d1/platform/alpha-migration.md
@@ -19,7 +19,7 @@ This guide will instruct you to recreate alpha D1 databases on our production-re
 ## Prerequisites
 
 1. You have the [`wrangler` command-line tool](/workers/wrangler/install-and-update/) installed
-2. You are using `wrangler` version `3.33.0` or later (released March 2024) as earlier versions do not have the [`--remote` flag](`/d1/platform/changelog/#change-in-wrangler-d1-execute-default) required as part of this guide
+2. You are using `wrangler` version `3.33.0` or later (released March 2024) as earlier versions do not have the [`--remote` flag](/d1/platform/changelog/#change-in-wrangler-d1-execute-default) required as part of this guide
 3. An 'alpha' D1 database. All databases created before July 27th, 2023 ([release notes](/d1/platform/changelog/#new-default-storage-subsystem)) use the alpha storage backend, which is no longer supported and was not recommended for production.
 
 ## 1. Verify that a database is alpha

